docs/tutorials/committing-liquidity.md

```diff
@@ -0,0 +1,143 @@
+* How to prepare and submit a liquidity commitment to express your desire to provide liquidity on a live market created with Vega
+* High-level tactics for placing orders to meet your commitment
+* How to change your commitment amount by amending and/or cancelling
+
+At the end of the tutorial, find troubleshooting tips as well as more resources for understanding the mechanics of liquidity provision on Vega. 
+
+## Rewarding liquidity providers
+Participants who provide enough liquidity to meet the [liquidity SLA](../concepts/liquidity/rewards-penalties.md#liquidity-sla) earn from liquidity fees, which are paid by the *price takers* on each trade. The SLA requires that you provide a specified amount of liquidity for a specified amount of time in order to receive any liquidity fee revenue.
+
+The percentage charged to each price taker is determined by sorting all the proposed fee factors. In your liquidity commitment submission, you need to propose a *fee factor*, as a decimalised number that is converted to percentage. 
+
+Your proposed fee factor is used, with other proposed fee factors, to determine the market's liquidity fee. Once the fee for the market is set, all liquidity orders charge that fee, regardless of whether the provider's submitted fee was higher or lower, and whether you propose a higher (or lower) factor, you are still a liquidity provider. The proposed fees are used to calculate the actual fee each participant will pay on a trade in that market.
+
+:::note Read more 
+[Proposing liquidity fee](../concepts/liquidity/rewards-penalties.md#determining-the-liquidity-fee-percentage)
+[Liquidity rewards](../concepts/liquidity/rewards-penalties.md)
+:::
+
+#### List available markets [WIP]
+You'll need to identify which market you want to supply liquidity to, and add the market ID to the submission transaction. 
+
+API LINK
+```bash
+```
+
+## Creating a liquidity commitment
+This tutorial describes how to create, amend or cancel, and send a liquidity commitment submission.
+
+**The liquidity commitment submission must include**:
+* The **market’s unique ID**, denoted as `marketId` - Confirm that the market is in a state to accept liquidity commitment, and is not a rejected market, has not had trading terminated, or has not been settled 
+* **Liquidity commitment amount**: The amount of funds that you want to allocate to providing liquidity. The amount will be moved into a bond account during the duration of your liquidity commitment, denoted as `commitmentAmount`
+* **Proposed liquidity fee**: The scaling factor for the fee you are bidding to receive when your order is matched, on a scale between 0 and 1. For example, a fee level of 0.01 would mean `0.01 * total trade amount` is charged. Note: Your proposed fee is used along with other proposed fees and commitments to determine the actual fee percentage for the market. [Learn how all proposed fee levels influence the market's fees]). Denoted as `fee` Denoted as `fee`
```

Comment:
Scale is between `0` and a network parameter `market.liquidity.maximumLiquidityFeeFactorLevel`.